conf/defaults.ini

```diff
@@ -868,6 +868,16 @@ max_annotations_to_keep =
 # Enable the Explore section
 enabled = true
 
+#################################### Help #############################
+[help]
+# Enable the Help section
+enabled = true
+
+#################################### Profile #############################
+[profile]
+# Enable the Profile menu
```

Comment:
```suggestion
# Enable the Profile section
```

I think it is worth of thinking on this like disabling the whole "profile" section of Grafana. Not only the menu item.